src/js/hooks/useSikkerhetsnivaa.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import Config from '../globalConfig';
+
+const useSikkerhetsnivaa = (brukernotifikasjon, type, innlogging) => {
+  const erMaskert = brukernotifikasjon.sikkerhetsnivaa === 4 && innlogging.securityLevel === '3';
```

Comment:
Den ene (4) er int, mens den er char (3). Skal det være likt?

Reply:
Dittnav-api returnerer int for `sikkerhetsnivaa`, men innloggingslinje-api returnerer string for `securityLevel`. Det kan være litt forvirrende, men koden over skal gi riktig visning :)

Reply:
Skulle vi gjort noe med dette @greger-nl  ?  Blir jo litt forvirrende.

Reply:
Ja, enig i at dette ser rart ut. Grunnen til at innloggingslinje-api returnerer en string er pga den gamle dekoratøren, den takler ikke int. Internt i innloggingslinje-api er det brukt en int, og det er bare i jackson-konverteringen til json at det er satt opp å mappe om fra int til string.
Der innlogging.securityLevel hentes inn, er det mulig å der gjøre om fra char til int? Slik at det bare er et sted at datatypene er ute av synk. Da bli logikken ryddig ellers i appen.

Reply:
Er ikke så glad i typekonvertering i javascript fordi det kan gi inkonsistente svar. Foreslår at vi lar det ligge, så kan vi kanskje se på om det kan fikses når den nye dekoratøren er ute?

Reply:
Ja, vi bør jo da fikse det slik at vi ikke returnerer ulike typer fra backend hvertfall 👍